### ISSUE TITLE
project-config/rebuild unsets graphql schema settings

### ISSUE THREAD

**Issue by riekusdn** [craftcms/cms#5961]
### Description
When I make changes to a graphql schema through the control panel, the changes are synced to project.yaml (both `schemas` and `scopes` are updated in the `graphql` section). However, when I run the `project-config/rebuild` command, only `scopes` is added to the project.yaml file, no `schemas`.

### Steps to reproduce

1. alter a schema through the control panel and save.
2. check the `graphql` section in project.yaml. It contains the `schemas` key and the `scopes` key
3. run `project-config/rebuild`
4. check the `graphql` section in project.yaml. It contains only the `scopes` key

### Additional info

- Craft version: 3.4.16
- PHP version: 7.2.29
- Database driver & version: MySQL 5.7.29
- Plugins & versions: Ether SEO 3.6.4

**Comment by andris-sevcenko** [comment]
Oops!

I just fixed this for the next release!

To get the fix early, change your `craftcms/cms` requirement in composer.json to:

```json
"require": {
  "craftcms/cms": "dev-develop#91ec317fa6f7f189aa6ca9f251e22fe5172c26d1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by brandonkelly** [comment]
Craft 3.4.17 is out now with this fix.

**Comment by chrisrowe** [comment]
This has reappeared in 3.5.0 beta-1

**Comment by chrisrowe** [comment]
oh wait, this was after than, nvm

**Comment by brandonkelly** [comment]
@chrisrowe It has since been merged into the `3.5` branch. You can update to the latest on that branch by changing your `craftcms/cms` requirement to `3.5.x-dev`, and then running `composer update`.
